src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -233,6 +233,13 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
     if (!item) return;
     const itemId = item.getId();
 
+    const customProps = customizeTableRowProps ? customizeTableRowProps(item) : {};
+
+    // make sure a disabled item is not in the selected state
+    if (customProps.disabled) {
+      store.unselect(item);
+    }
```

Comment:
I'm not sure we need these lines. 
1. It doesn't sound right to make side effects (unselecting from store) inside `getRow` prop which should have only 1 rendering purpose.
2. Selection/unselection should be happened from specific places only (checkboxes).
3. It's unclear when this is needed. While testing I didn't spot any changes with or without these lines.

Is there any specific case, where such unselection needed?

Reply:
My concern was that if a user "selects all" and there is a disabled item, what happens if that disabled item becomes enabled? I see now this should just result in clearing the "select all" checkbox so no need for this bulletproofing (which I don't think was right anyway :( ). I'll do some more tests...